bound/kt/src/main/kotlin/web5/sdk/vc/VerifiableCredential.kt

```diff
@@ -75,6 +115,34 @@ data class VerifiableCredential private constructor(
             )
         }
 
+        /**
+         * Creates a new Verifiable Credential with the specified issuer, subject, and optional creation options.
+         *
+         * @param issuer The entity issuing the credential. The `issuer` must be a valid DID.
+         * @param credentialSubject The subject of the credential containing claims. The subject must be a valid DID.
+         * @param options Optional parameters for creating the credential, such as schema or status.
+         * @return The newly created Verifiable Credential.
+         * @throws Web5Exception if the creation fails due to validation or other errors.
+         *
+         * Example usage:
+         * ```
+         * val issuerBearerDid = DidJwk.create()
+         * val subjectDidUri = "did:dht:ng4hmqtrgujox4agpf8okxihnyy1zqnq97qfeq15x8oar7yepzhy"
+         * val verifiableCredential = VerifiableCredential.create(
+         *     Issuer.StringIssuer(issuerBearerDid.did.uri),
+         *     CredentialSubject(id = subjectDidUri),
+         *     VerifiableCredentialCreateOptions(
+         *         credentialStatus = CredentialStatus(
+         *             id = "https://example.com/status/1",
+         *             type = "StatusList2021Entry",
+         *             statusPurpose = "revocation",
+         *             statusListIndex = "3",
+         *             statusListCredential = "https://example.com/status/1"
```

Comment:
```suggestion
```

could take this ouut and just show a default vc instead of one with credential status